src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -118,7 +118,15 @@ protected internal override void ClosePin(int pinNumber)
                 try
                 {
                     SetPinEventsToDetect(pinNumber, PinEventTypes.None);
-                    File.WriteAllText(Path.Combine(GpioBasePath, "unexport"), pinOffset.ToString(CultureInfo.InvariantCulture));
+                    try
```

Comment:
can you please split PR into bug fixes from new APIs? otherwise if anything causes regression we will have to undo the whole change

Reply:
I created two new branches on my fork with the changes separated.
Is it possible to update this PR or do I have to create new PRs?

Reply:
You can update a PR by committing to the same branch again (you can force-push, if required). Create a new PR for the second change.